core/src/main/java/org/pentaho/platform/workitem/WorkItemLifecyclePhase.java

```diff
@@ -46,6 +46,14 @@
    * The work item execution is in progress
    */
   IN_PROGRESS,
+  /**
+   * The work item execution is in progress
+   */
+  COMPLETED,
```

Comment:
what's the difference between `COMPLETED` and `SUCCEEDED` , a few lines below? Is the latter one being used anywhere? If not, we could clean it, to remove confusions

Reply:
It's been used by ActionRunner.